docs/guides/upgrade_paths_for_neon_os_v2.md

```diff
@@ -0,0 +1,15 @@
+# Ugrade Paths for Neon OS Version 2.0 on the Mark II
+Describes your basic options for upgrading your Neon OS to V2.0.
+
+## Options
```

Comment:
This doesn't follow the formatting of other guides with `Steps`; maybe this belongs in some other section of the docs or a page on the website?